anvil/src/winit.rs

```diff
@@ -88,21 +89,18 @@ impl Backend for WinitData {
     fn early_import(&mut self, _surface: &wl_surface::WlSurface) {}
 }
 
-#[cfg(feature = "debug")]
+#[cfg(not(feature = "debug"))]
 smithay::backend::renderer::output::element::render_elements! {
-    pub CustomRenderElements<'a, R>;
-    Surface=smithay::backend::renderer::output::element::surface::WaylandSurfaceRenderElement,
-    Texture=smithay::backend::renderer::output::element::texture::TextureRenderElement<<R as Renderer>::TextureId>,
-    Fps=&'a FpsElement<<R as Renderer>::TextureId>
+    pub CustomRenderElements<R>;
+    Pointer=PointerRenderElement<R>,
+    Surface=WaylandSurfaceRenderElement,
 }
-
-smithay::desktop::space::space_elements! {
-    CustomSpaceElements<'a, R>[
-        WaylandSurfaceRenderElement,
-        TextureRenderElement<<R as Renderer>::TextureId>,
-    ];
-    Pointer=&'a PointerElement<<R as Renderer>::TextureId>,
-    SurfaceTree=SurfaceTree,
+#[cfg(feature = "debug")]
+smithay::backend::renderer::output::element::render_elements! {
+    pub CustomRenderElements<R>;
+    Pointer=PointerRenderElement<R>,
+    Surface=WaylandSurfaceRenderElement,
+    Fps=FpsElement<<R as Renderer>::TextureId>,
```

Comment:
This could be guarded with a cfg attribute instead of duplicating the declaration?

Reply:
Currently the macro doesn't allow that it seems, but it is on my todo-list

Reply:
Yes, but I have opted to use `SpaceRenderElements` in case you do not need custom render elements, just to show-case both possibilities.